doc/userguide/devguide/internals/engines/stream/applayertriggerrawreassembly.rst

```diff
@@ -0,0 +1,37 @@
+AppLayerParserTriggerRawStreamReassembly
```

Comment:
Maybe here we could have a more human-friendly title, and in the section we show the actual function name(s)?